docs/fides/docs/installation/configuration.md

```diff
@@ -0,0 +1,37 @@
+# Configuration
+
+Fidesctl supports two methods of configuration. The first is via a toml file, and the second is via environment variables. They can also be used in tandem, with the environment variables overriding the toml configuration values.
+
+By default the fidesctl CLI doesn't require a config file and will instead leverage the default values. These are very likely to be wrong however so it is recommended to always configure your settings properly.
+
+## Configuration file
+
+Here's an example of a fidesctl configuration file:
+
+```toml
+[cli]
+server_url = "http://localhost:8080"
+
+[api]
+database_url = "postgresql://postgres:postgres@localhost:5432/fidesctl"
+```
+
+By default fidesctl will look for a configuration file in the following three places:
+
+1. At the path specified by the `FIDESCTL_CONFIG_PATH` environment variable
```

Comment:
Just checking, is it correct to only have one underscore separating the env var name elements?

Below, we call out how there are two underscores for fidesctl env vars, so it seems odd.

Reply:
This is not overriding a value in the config file, therefore it doesn't follow the same pattern